website/static/js/projectorganizer.js

```diff
@@ -417,8 +421,8 @@ function _showProjectDetails(event, item, col) {
                 };
             postAction = $osf.postJSON(url, postData);
             postAction.done(function () {
-                treebeard.updateFolder(null, theParentNode);
-                triggerClickOnItem.call(treebeard, item);    
+                treebeard.updateFolder(null, treebeard.find(1));
```

Comment:
I see. Sorry, I missed that part in the description. 

@brianjgeiger Did you intentionally not update the whole grid on rename?